port/jemalloc_helper.h

```diff
@@ -30,7 +30,7 @@ extern "C" int mallctlbymib(const size_t*, size_t, void*, size_t*, void*,
 extern "C" void malloc_stats_print(void (*)(void*, const char*), void*,
                                    const char*) __attribute__((__weak__));
 extern "C" size_t malloc_usable_size(JEMALLOC_USABLE_SIZE_CONST void*)
-    JEMALLOC_CXX_THROW __attribute__((__weak__));
```

Comment:
Since I don't know how these are used by jemalloc, at a first glance removing these may not be the right fix. Should we ifdef JEMALLOC_CXX_THROW and malloc_usable_size on your build platform?

Reply:
I have no idea I am afraid, but we can't even build from source code on Ubuntu 18 or CentOS 6 without these changes.